tensorflow/python/kernel_tests/random/random_ops_test.py

```diff
@@ -30,6 +30,11 @@
 from tensorflow.python.platform import test
 
 
+FLOATS = [dtypes.float16, dtypes.float32, dtypes.float64]
+if test_util.is_gpu_available(
+      cuda_only=True, min_cuda_compute_capability=(8, 0)):
```

Comment:
Unfortunately, calling `is_gpu_available` before `test.main()` is not allowed in the internal version of TF. Note `test.main()` is called at the bottom of the file to run all the tests.

To fix, I would create a function that returns these floating-point types, maybe called `get_float_types`, and call it everywhere you currently use `FLOATS`.

Reply:
Got it, that sounds good to me.